packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/authcodepkce/examples.md

```diff
@@ -0,0 +1,13 @@
+### Examples
+
+The following native and single-page application examples show the Authorization Code with PKCE flow, as it would be implemented by an application that needs to authenticate a user. These are complete example applications that show the entire experience.
```

Comment:
```suggestion
The following native and single-page application examples show the Authorization Code with PKCE flow as it would be implemented by an application that needs to authenticate a user. These are complete example applications that show the entire experience.
```